analyzer/src/funTest/kotlin/managers/Extensions.kt

```diff
@@ -26,17 +26,35 @@ import io.kotest.matchers.should
 import java.io.File
 
 import org.ossreviewtoolkit.analyzer.PackageManager
+import org.ossreviewtoolkit.analyzer.PackageManagerResult
+import org.ossreviewtoolkit.model.DependencyGraph
 import org.ossreviewtoolkit.model.ProjectAnalyzerResult
 import org.ossreviewtoolkit.model.yamlMapper
 
 fun Any?.toYaml() = yamlMapper.writeValueAsString(this)!!
 
-fun PackageManager.resolveSingleProject(definitionFile: File, resolveScopes: Boolean = false): ProjectAnalyzerResult =
-    resolveDependencies(listOf(definitionFile)).projectResults[definitionFile].let { resultList ->
+fun PackageManager.resolveSingleProject(definitionFile: File, resolveScopes: Boolean = false): ProjectAnalyzerResult {
```

Comment:
The refactoring of `resolveSingleProject` seems unrelated to the change and should have been done in a separate commit.

Reply:
I think, this has to be done in the same commit together with the changes on the test classes; otherwise, tests will fail.